packages/addons/storybook/index.ts

```diff
@@ -12,7 +14,12 @@ export default defineAddon({
 		runsAfter('eslint');
 	},
 	run: async ({ sv }) => {
-		const args = ['create-storybook@latest', '--skip-install', '--no-dev'];
+		const args = [
+			`create-storybook@${STORYBOOK_VERSION}`,
+			'--skip-install',
+			'--no-dev',
+			'--no-features'
```

Comment:
I don't think this should be in production, I think users should be able to choose the feature they want. Storybook will prompt them when this flag is not set.

When we discussed this, I thought you only put in the flag in your tests, to speed them up.